### PR TITLE
Update packages to latest versions

### PR DESCRIPTION
**Pull request by abtris**
- Update packages to latest
- Update compatibility with V3 iron_mq - https://github.com/iron-io/iron_mq_node

### CODE REVIEW

**Comment by abtris** [comment]
@Almad can you look at this PR, it's some breaking changes with new iron_mq node package

File: src/queues/ironmq.coffee [hunk 1]
```diff
@@ -139,20 +140,23 @@ class IronMQQueue
...
     catch e
       logger.error 'ironTask is', ironTask
       logger.error "IVY_IRONMQ_ERROR Retrieve tasks that cannot be parsed as JSON, deleting from queue: #{ironTask}", e
-      @getQueue().del ironTask.id, (err, body) =>
-        if err
-          logger.warn "IVY_WARNING Cannot delete task from IronMQ", err
-          @manager.emit 'mqError', err
+      logger.warn "IRONTASK", ironTask
+      if ironTask.id
+        @getQueue().del ironTask.id, { reservation_id: taskArgs.options.reservation_id}, (err, body) =>
```
Comment (Almad) [inline src/queues/ironmq.coffee]: nit: delete space after `{`

**Comment by Almad** [comment]
IMHO in sync with the newest version, looks good to me™

**Comment by abtris** [comment]
Here is doucumentation for migration http://dev.iron.io/mq/v2_v3_migration_guide/  I think we cover all things.

**Pull request merged**